iOS14UIKitTour/iOS14UIKitTour/API/GithubEndpoints.swift

```diff
@@ -0,0 +1,24 @@
+
+import Foundation
+
+/// Github Endpoint manager
+enum GitHubEndpoints {
+    // All Swift Miami Repositories
+    case repositories
+    
+    // Collaborators for a given Repository
+    case contributors(String)
+    
+    // Non-optional URL
+    var url: URL {
+        switch self {
+        case .repositories:
+            return URL(string: "https://api.github.com/orgs/swift-miami/repos")!
+        case .contributors(let repository):
+           
+            let ur = URL(string: "https://api.github.com/repos/swift-miami/\(repository)/contributors")!
+            print(ur)
+            return ur
```

Comment:
Please remove print statements from the code. It's fine if you like to use them to debug but clean up before submitting a PR.

Also, please be consistent, either both case statements do `let url = ...` or both `return URL(...`. 

Finally, remove empty spaces.